# Generated from: /home/latty/link_dir/2015_li3ds/__ROS__/Ellipse_N/ROS/overlay_ws/src/sbg_ros_driver/sbgECom/src/binaryLogs/sbgEComBinaryLogUsbl.h
# SbgLogUsblData
uint32 timeStamp
uint16 status
float64 latitude
float64 longitude
float32 depth
float32 latitudeAccuracy
float32 longitudeAccuracy
float32 depthAccuracy